src/modules/cache/CacheService.ts

```diff
@@ -33,4 +33,21 @@ export class CacheService extends DynamicHostService {
       ids,
     )
   }
+
```

Comment:
This is not required. Content block templates will be fetched using listItems and getItems request handler only